fv3core/stencils/temperature_adjust.py

```diff
@@ -17,6 +14,20 @@ def compute_pkz_tempadjust(
     pkz: FloatField,
     delt_time_factor: float,
 ):
+    """
+    Adjust air temperature from heating due to vorticity damping.
```

Comment:
you might consider putting this in the same file as dyn_core, but Iwon't object to leaving it here for now.

Reply:
scratch that, just saw the similar conservation you had with Eddie in the pe_halo PR. let's leave it for now.

Reply:
Thanks for reviewing this!